### PR TITLE
Return distinct info about service

### PR DESCRIPTION
**Pull request by jsaalfeld**

### CODE REVIEW

**Comment by jsaalfeld** [comment]
This later will include https://github.com/gdi-by/downloadclient-test/issues/38#issuecomment-244867617

**Comment by jsaalfeld** [comment]
The feature was included

File: src/main/java/de/bayern/gdi/gui/Controller.java [hunk 1]
```diff
@@ -274,46 +274,71 @@ private void clearUserNamePassword() {
...
+                                    I18n.format("status.ready"));
+                        }
+                    } catch (IOException e) {
+                        log.log(Level.SEVERE, e.getMessage(), url);
+                    } finally {
+                        serviceURL.getScene().setCursor(Cursor.DEFAULT);
+                        serviceSelection.setDisable(false);
+                    }
+                };
+                //task.run();
```
Comment (s-l-teichmann) [inline src/main/java/de/bayern/gdi/gui/Controller.java]: remove this line

Comment (jsaalfeld) [reply src/main/java/de/bayern/gdi/gui/Controller.java]: Done

**Comment by dmth** [comment]
HI, that's not working correctly. Try: Search for Haus, selected "Pre-Defined Hütten und ..." now use "Dienst wählen", wait until the service is loaded. Now paste a URL of a WFS: "http://geoserv.weichand.de:8080/geoserver/wfs" whcih should be corrected by the new URL-Guesser. Unfortuately the Atom service is still displayed.

**Comment by dmth** [comment]
That's better now: next issue: If you access a password-protected service and enter username and password, and switch to a different service, the username and password field are greyed out but they are not cleared....

I'd expect that information on the username and password is also removed in this case.

**Comment by dmth** [comment]
better. works as expected.

**Pull request merged**